pages/[slug].tsx

```diff
@@ -26,12 +26,14 @@ export const getStaticProps: GetStaticProps<ArticleProps> = async context => {
     const collections = await loadRecordCollections('posts')
     const { recordCollections } = collections
     const parentCollection = recordCollections.find(collection => !!collection.members.find(member => member.slug === slug))
-    const recordAuthor = markdownFile.frontMatter.author ? slugToTitleCase(markdownFile.frontMatter.author) : null
+    const recordAuthorId = markdownFile.frontMatter.author ?? null
```

Comment:
Instead of creating two variables here, can we pass the `slug` as `author` (calling it something like `authorSlug`), use it in the link, and then use `slugToTitleCase` in the template for presentation purposes?